controllers/addPublisher.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+//$database = require 'core/bootstrap.php';
+//
+//$newPublisher = $app['database']->insertPublisher('publishers', [
+//    'name' => $_POST['newPublisher'],
+//]);
+
```

Comment:
Su codesnifferiu isvengsi ateiyje, taciau is auksine taisykle - ne daugiau vieno tarpo tarp eiluciu.